server/routes/request.ts

```diff
@@ -17,6 +17,9 @@ requestRoutes.get('/', async (req, res, next) => {
   try {
     const pageSize = req.query.take ? Number(req.query.take) : 10;
     const skip = req.query.skip ? Number(req.query.skip) : 0;
+    const requestedBy = req.query.requestedBy
+      ? Number(req.query.requestedBy)
+      : 0;
```

Comment:
Any reason we don't just set it to null here if there isn't a query value?

Reply:
Will change that! 👍🏻